BackEndApp/authv1/views.py

```diff
@@ -11,32 +12,51 @@ def login(request):
 
     username = request.data.get('username')
     password = request.data.get('password')
-
     user = User(username, password)
     user = user.find()
-    token = user.get('token', 'asdfghjklkjhgfdsa')
+
+    if user is None or not bcrypt.checkpw( password.encode('utf-8'),
+                                           user.get('password')
+                                          ):
+        status = 401
+        message = "Invalid credentials"
+        token = None
+    else:
+        status = 200
+        message = "Login Successful"
+        token = user.get('token', 'asdfghjklkjhgfdsa')
+    # TODO: Implement token based authentication
+
     return JsonResponse({
-                            'message': "Login Successful", 
-                            'status': 200,
+                            'message': message, 
+                            'status': status,
                             'user': username,
                             'token': token
                         })
 
 
 @api_view(['POST'])
 def register(request):
+    try:
+        username = request.data.get('username')
+        email = request.data.get('email')
+        password = request.data.get('password')
+        first_name = request.data.get('first_name')
+        last_name = request.data.get('last_name')
 
-    username = request.data.get('username')
-    email = request.data.get('email')
-    password = request.data.get('password')
-    first_name = request.data.get('first_name')
-    last_name = request.data.get('last_name')
 
-    user = User(username, password, **{'email': email, "first_name": first_name, "last_name": last_name})
-    user_id = user.create()
-    
+        user = User(username, password, **{'email': email, "first_name": first_name, "last_name": last_name})
+        user_id = user.create()
+
+        message = "Registered Successfully"
+        status = 200
+
```

Comment:
```suggestion
        token = user.get('token', 'asdfghjklkjhgfdsa')
```

As discussed, we'll login right after registration, better add dummy token in this PR itself